.github/workflows/package.yml

```diff
@@ -7,6 +7,32 @@ on:
   workflow_dispatch:
 
 jobs:
+  build_windows:
+    name: build on windows-2022
+    runs-on: windows-2022
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: ucrt64
+          install: git make
+          pacboy: cmake gcc pkgconf
+      - run: |
+          git clone https://github.com/libgit2/libgit2.git
+          mkdir libgit2/build
+          cd libgit2/build
+          cmake .. -DCMAKE_INSTALL_PREFIX='C:/ProgramData/libgit2' -DBUILD_CLI=OFF -DBUILD_SHARED_LIBS=OFF -DBUILD_TESTS=OFF
+          cmake --build . --target install --parallel 4
```

Comment:
#### Important

Turn this into a shell script. Put the command to compile statically in it rather than in `Makefile`.

Reply:
Done.